keyboards/yowkees/keyball39/rules.mk

```
@@ -0,0 +1,9 @@
+# Optical sensor driver for trackball.
+POINTING_DEVICE_DRIVER = custom
+SRC += drivers/pmw3360/pmw3360.c
+QUANTUM_LIB_SRC += spi_master.c # Optical sensor use SPI to communicate
```

Honestly, I would prefer that the custom code be removed and the core pmw3360 code be used.  

Though, the sroms should probably be removed, and shouldn't be necessary. 

But at the very least:
```suggestion
SRC += drivers/pmw3360/pmw3360.c
SPI_DRIVER_REQUIRED = yes
```

(custom is the default driver, and spi-driver-required handles the spi includes)

---

Question for my own education: 

What's the difference between following your Suggested Changes versus setting `POINTING_DEVICE_DRIVER = pmw3360` (which is what my referenced other branch does)?

---

> custom is the default driver

Near as I can tell, [this is actually not the case](https://github.com/qmk/qmk_firmware/blob/8253697a6389aaec73f85a29dd25247b32389502/builddefs/common_features.mk#L123-L162), and setting `POINTING_DEVICE_DRIVER = custom` is required.

Using `SPI_DRIVER_REQUIRED` seems fine.

I am still investigating switching to the core 3360 driver. My most recent attempt results in oversize firmware (approximately 31 KB), which I don't remember having as an issue on my other branch.

---

Would be willing to help as well, however I use the keyball44 variant. I know how to code, however very green with contributing to github projects. please let me know!